### PR TITLE
Bug 1254061, added note on whitelisting users

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1254061

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@liggitt @rjhowe PTAL at my first stab

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -550,6 +547,12 @@ configured URL. If empty, system trusted roots are used. Only applies if
 `ldaps://` URLs connect using TLS, and `ldap://` URLs are upgraded to TLS.
 <13> An RFC 2255 URL which specifies the LDAP host and search parameters to use,
 xref:ldap-url[as described above].
+
+[NOTE]
+====
+To whitelist users for an LDAP integration, use the `lookup` mapping method.
+Before a login from LDAP would be allowed, a cluster administrator must create
+an identity and user object.
```
Comment (liggitt) [inline install_config/configuring_authentication.adoc]: ...an identity and user object for each LDAP user.

**Comment by liggitt** [comment]
nit, LGTM

**Comment by ahardin-rh** [comment]
@liggitt Thanks! Updated.

@openshift/team-documentation Please peer review

**Comment by mburke5678** [comment]
LGTM. Would it make sense to put the note right under #4 where it talks about the mapping method parameter??

**Comment by ahardin-rh** [comment]
Approved by QE

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[Configuring Authentication and User Agent]
|Added a note about whitelisting users.
%

**Pull request merged**